pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -32,48 +50,43 @@ import (
 // deployments will be cancelled. The controller will not attempt to scale
 // running deployments.
 type DeploymentConfigController struct {
-	// kubeClient provides acceess to Kube resources.
-	kubeClient kclient.Interface
-	// osClient provides access to OpenShift resources.
-	osClient osclient.Interface
+	// oc provides access to OpenShift resources.
+	oc osclient.Interface
+	// kc provides acceess to Kube resources.
+	kc kclient.Interface
+
+	// queue contains deployment configs that need to be synced.
+	queue *workqueue.Type
```

Comment:
you probably want a `workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter())` and `workqueue.RateLimitingInterface`